### ISSUE TITLE
latest/polaris.css doesn't have styles for ResourceList.Item

### ISSUE THREAD

**Issue by vladimir-e** [Shopify/polaris#258]
# Issue summary

This is how I include polaris.css to my project:

```
<link rel="stylesheet" href="https://sdks.shopifycdn.com/polaris/latest/polaris.css">
```

But since today ResourceList items don't have styles and they are not clickable.

However if I specify the version (today's latest `1.9.1`):
```
<link rel="stylesheet" href="https://sdks.shopifycdn.com/polaris/1.9.1/polaris.css">
```

styles work as expected. 

So current `latest/polaris.css` is missing some CSS.

## Expected behavior

Before today:

![prodsync app 2018-01-18 12-51-43](https://user-images.githubusercontent.com/1533802/35081547-d0e974f0-fc4f-11e7-9255-c380486668e2.png)

## Actual behavior

Today:

![prodsync app 2018-01-18 12-52-03](https://user-images.githubusercontent.com/1533802/35081563-e67d679a-fc4f-11e7-8cd0-a6253a9453e7.png)

## Steps to reproduce the problem

```
<link rel="stylesheet" href="https://sdks.shopifycdn.com/polaris/latest/polaris.css">
```

```
<ResourceList
  items={this.props.feeds}
  renderItem={this.renderDataFeed}
/>
//....
renderDataFeed(item, index) {
  return (
    <ResourceList.Item
      key={index}
      url={`/feed/${item.id}`}
      attributeOne={item.title}
      attributeTwo={<TextStyle variation="subdued">No products</TextStyle>}
      persistActions={true}
    />
  );
}
```

## Specifications

- Polaris version: 1.9.1
- React version: 16.0.0
- Browser: Safari Version 11.0.2 (13604.4.7.1.6)
- Device: Macbook pro
- Operating System: MacOS

**Comment by kaelig** [comment]
Thank you very much for bringing this up to our attention, @vladimir-e.

It seems our CDN has picked up the latest beta version (`2.0.0-beta.4`) instead of the latest stable (`1.9.1`). We're investigating and will restore it to the right version asap.

The current workaround is, as you found out, to specify the actual version in the URL, like so:

```html
<link rel="stylesheet" href="https://sdks.shopifycdn.com/polaris/1.9.1/polaris.min.css" />
```

**Comment by dleroux** [comment]
Hi @vladimir-e this issue is now fixed in release of 1.10. Thanks for raising it to us!

**Comment by vladimir-e** [comment]
Thanks for the update!
